src/map/Map.js

```diff
@@ -624,6 +627,16 @@ L.Map = L.Evented.extend({
 		this._container.scrollLeft = 0;
 	},
 
+	_onMoveEnd: function () {
+		if (L.Browser.any3d && this.options.transform3DLimit &&
+			(Math.abs(this._mapPane._leaflet_pos.x) >= this.options.transform3DLimit
+			 || Math.abs(this._mapPane._leaflet_pos.y) >= this.options.transform3DLimit)) {

```

Comment:
There's a private method `_getMapPanePos`, let's use it. Also, you could make this shorter if you use `Math.max`.